src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,52 @@
+def time_series_explorer(input_df: pd.DataFrame, 

```

Comment:
I think we should be consistent across our function naming! I saw that @phchen5's function is called `plot_binary` so maybe we could have all four follow this `plot_...` naming convention?

```suggestion
def plot_timeseries(input_df: pd.DataFrame, 

```

Reply:
I agree, let's discuss about that naming convention first!